### ISSUE TITLE
Got error when `wails dev`

### ISSUE THREAD

**Issue by choengchaip** [wailsapp/wails#3696]
### Description

I can run at normally but when I reinstalled windows I got the error

### To Reproduce

1. wails dev
2. got error after compile application

### Expected behaviour

It should open a windows application
![image](https://github.com/user-attachments/assets/71a66ba5-fddf-4689-9a70-d1f47889a362)

### Screenshots

![1](https://github.com/user-attachments/assets/e4ba5d11-ea1b-4f27-b231-69386965d8b5)
![2](https://github.com/user-attachments/assets/20f7d7c6-edcf-4534-afd9-ee2e16922f91)
![3](https://github.com/user-attachments/assets/a82ed57a-b4fc-405a-a77e-36cab29901df)

### System Details

```shell
| Dependency | Package Name | Status    | Version        |
| WebView2   | N/A          | Installed | 127.0.2651.105 |
| Nodejs     | N/A          | Installed | 20.16.0        |
| npm        | N/A          | Installed | 10.8.1         |
| *upx       | N/A          | Available |                |
| *nsis      | N/A          | Available |                |
```

**Comment by choengchaip** [comment]
Fixed by downgrade to 2.8.2

**Comment by choengchaip** [comment]
Fixed by downgrade to 2.8.2
